content/docs/concepts/multicluster-deployments/index.md

```diff
@@ -12,39 +12,94 @@ a single controllable composite application.
+
+{{< image width="80%"
+    link="./single-mesh-or-mesh-federation.png"
+    caption="Decision tree for selecting single multicluster mesh or mesh federation"
+>}}
 
 ## Multicluster service mesh
 
-A multicluster service mesh is a mesh composed of services running within more than one underlying
-cluster but with all services running under a single administrative control.
-In a multicluster service mesh, a service named `foo` in namespace `ns1` of cluster 1 is the same service
-as `foo` in `ns1` of cluster 2.
-This is different from a loosely-coupled service mesh where two clusters may have different
-definitions of the same service which will need to be reconciled when integrating the clusters.
+A multicluster service mesh is a **single logical** mesh composed of services running in multiple clusters.
+A multicluster service mesh has the following characteristics:
+
+- {{< gloss >}}Uniform naming{{< /gloss >}}.
+- The combined clusters expose all their services to each other.
+- The traffic from the combined clusters is allowed to enter each of the clusters.
+- Common CA root between the Citadels in all the clusters.
+
+To learn more, see [single multicluster service mesh or mesh federation?](#single-multicluster-service-mesh-or-mesh-federation).
 
 A multicluster service mesh has the advantage that all the services look the same to clients,
 regardless of where the workloads are actually running. It's transparent
 to the application whether it's deployed in a single or multicluster service mesh.
 To achieve this behavior, a single logical control plane needs to manage all services,
 however, the single logical control plane doesn't necessarily need to be a single physical
-Istio control plane. There are two possible deployment approaches:
+Istio control plane. There are three possible deployment approaches:
+
+1. Multiple dedicated Istio control planes that have replicated service and routing configurations.
 
-1. Multiple Istio control planes that have replicated service and routing configurations.
+1. A shared Istio control plane that can access and configure the services in more than one cluster. The services are
+   directly reachable from all the workloads in all the clusters.
 
-1. A shared Istio control plane that can access and configure the services in more than one cluster.
+1. The same as the previous approach, but with services reachable through connected gateways.
 
-Even with these two approaches, there is more than one way to configure a multicluster service mesh.
+To select the correct topology, ask yourself the following questions:
+
+1. Do you require **a single control plane?**. Managing multiple Istio control planes
+  requires different infrastructure capabilities to those needed to manage a single
+  control plane. Be certain of your preference when planning your topology.
+
+1. Are you allowed by the security procedures in your organization to provide **access to Kubernetes APIs** of one
```

Comment:
allowed/permitted throughout document

Reply:
Why is permitted better than allowed? I checked PCI DSS v3.2.1 compliance document, allow appears on 40 pages, permit on 10 pages.

Some examples:
Allow:
_1.3.4 Do not allow unauthorized outbound traffic from the cardholder data environment to the Internet._
_1.1.6.b Identify insecure services, protocols, and ports allowed; and verify that security features are documented for each service._

Permit:
_1.3.5 Permit only “established” connections into the network._
_User interfaces that do not permit access to unauthorized functions._

Reply:
ya - makes sense.  thanks.